aiogram/types/base.py

```diff
@@ -241,7 +244,7 @@ def __iter__(self):
         for item in self.to_python().items():
             yield item
 
-    def iter_keys(self):
+    def iter_keys(self) -> T:
```

Comment:
Why here you use T? iter_keys is generator so use generator/iterator annotation like iter_values